app/models/concerns/relatable.rb

```diff
@@ -28,6 +28,8 @@ def self.get_related_model(counter_name)
       counter_name = counter_name.split(/ as /i)[-1]  # honor aliasing
       counter_name.gsub!('_count','')   # support cached count columns as well
 
+      counter_name.gsub!('qtls','qtl')  # unfortunate special case... :(
```

Comment:
@kammerer "We changed a lot, so why not this?" is a non sequitur. The fact that we introduce a lot of changes is not an argument in favor of any particular change. Frankly, I was unaware that this was such a big headache for the upper layers of the stack. I am happy to rename the table even now, if you so desire.

You are right in that acronyms are sometimes pluralized in the way cited.